go.mod

```diff
@@ -3,10 +3,12 @@ module github.com/celestiaorg/celestia-node
 go 1.16
 
 require (
+	github.com/celestiaorg/celestia-core v0.0.1-mvp-das-lightclient.0.20210831102741-321cdc4f8dc4
```

Comment:
This should be updated as soon as either @evan-forbes or @marbar3778 updated core to the latest tendermint. Although I doubt the rpc client changed much anyways. Just something to keep in mind.